sql/hive/benchmarks/OrcReadBenchmark-results.txt

```diff
@@ -2,155 +2,155 @@
+Native ORC MR                                      3347           3376          42          3.1         319.2       1.0X
+Native ORC Vectorized                              1220           1225           7          8.6         116.3       2.7X
+Hive built-in ORC                                  4168           4184          23          2.5         397.5       0.8X
 
-OpenJDK 64-Bit Server VM 1.8.0_222-b10 on Linux 3.10.0-862.3.2.el7.x86_64
+OpenJDK 64-Bit Server VM 1.8.0_232-8u232-b09-0ubuntu1~18.04.1-b09 on Linux 4.15.0-1044-aws
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 String with Nulls Scan (95.0%):           Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                      1912           1917           6          5.5         182.4       1.0X
-Native ORC Vectorized                               477            484           5         22.0          45.5       4.0X
-Hive built-in ORC                                  2374           2386          17          4.4         226.4       0.8X
+Native ORC MR                                      1851           1862          16          5.7         176.5       1.0X
+Native ORC Vectorized                               466            471           7         22.5          44.4       4.0X
+Hive built-in ORC                                  2523           2529           8          4.2         240.6       0.7X
 
 
 ================================================================================================
 Single Column Scan From Wide Columns
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_222-b10 on Linux 3.10.0-862.3.2.el7.x86_64
+OpenJDK 64-Bit Server VM 1.8.0_232-8u232-b09-0ubuntu1~18.04.1-b09 on Linux 4.15.0-1044-aws
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 Single Column Scan from 100 columns:      Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                       290            350         102          3.6         276.1       1.0X
-Native ORC Vectorized                               155            166          15          6.7         148.2       1.9X
-Hive built-in ORC                                   520            531           8          2.0         495.8       0.6X
+Native ORC MR                                       250            264          15          4.2         238.1       1.0X
+Native ORC Vectorized                               121            138          24          8.7         115.5       2.1X
+Hive built-in ORC                                  1761           1792          43          0.6        1679.3       0.1X
```

Comment:
Here there is a regression ~3 times. https://issues.apache.org/jira/browse/SPARK-30565